docker/Dockerfile

```diff
@@ -70,7 +70,8 @@ RUN cd /torchfort && mkdir build && cd build && \
     cmake -DCMAKE_INSTALL_PREFIX=/opt/torchfort \
     -DTORCHFORT_YAML_CPP_ROOT=/opt/yaml-cpp \
     -DTORCHFORT_NCCL_ROOT=/opt/nccl/build \
-    -DTORCHFORT_BUILD_EXAMPLES=1 \
+    -DTORCHFORT_BUILD_EXAMPLES=0 \
```

Comment:
Did you intend to disable building examples?

Reply:
I am sorry, I think I disabled that during the debug phase. I have enabled it again.